frontend/src/components/api_datacommponent/api_data.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+
+const ApiDataComponent = () => {
+    const [data, setData] = useState([]);
+    
+    useEffect(() => {
+        fetch('http://localhost:9123/api/get')
+          .then(response => response.json())
+          .then(data => {
+            console.log(data)
```

Comment:
Console log can be removed from the code

Reply:
Done.